storage/migrations/25_gasoutputs_height_code.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import "github.com/go-pg/migrations/v8"
+
+// Schema version 22 adds Height and ActorName to gas outputs table
+
+func init() {
+	up := batch(`
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN height bigint NOT NULL;
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN actor_name text NOT NULL;
+	ALTER TABLE public.derived_gas_outputs DROP CONSTRAINT derived_gas_outputs_pkey;
+	ALTER TABLE public.derived_gas_outputs ADD PRIMARY KEY (cid, height);
+`)
```

Comment:
@iand can shall we leave like this, or add state_root too?

Reply:
let's add state_root - I'll do it